docs/user_guides/projects/airflow/airflow.md

```diff
@@ -0,0 +1,99 @@
+
+### Use Apache Airflow in Hopsworks 
+
+Hopsworks deployments include a deployment of Apache Airflow. You can access it from the Hopsworks UI by clicking on the _Airflow_ button on the left menu.
+
+Airfow is configured to enforce Role Based Access Control (RBAC) to the Airflow DAGs. Admin users on Hopsworks have access to all the DAGs in the deployment. Regular users can access all the DAGs of the projects they are member of.
+
+!!! note "Access Control"
+    Airflow does not have any knowledge of the Hopsworks project you are currently working on. As such, when opening the Airflow UI, you will see all the DAGs of the projects you are member of.
+
+#### Hopsworks DAG Builder 
+
+<figure>
+  <img src="../../../../assets/images/guides/airflow/airflow_dag_builder.png" alt="Airflow DAG Builder"/>
+  <figcaption>Airflow DAG Builder</figcaption>
+</figure>
+
+You can create a new Airflow DAG to orchestrate jobs using the Hopsworks DAG builder tool. Click on _New Workflow_ to create a new Airflow DAG. You should provide a name for the DAG as well as a schedule interval. You can define the schedule using the dropdown menus or by providing a cron expression.
+
+You can add to the DAG Hopsworks operators and sensors:
+
+- **Operator**: The operator is used to trigger a job execution. When configuring the operator you select the job you want to execute and you can optionally provide execution arguments. You can decide whether or not the operator should wait for the execution to be done. If you select the _wait_ option, the operator will block and Airflow will not execute any parallel task. In this case the Airflow task fails if the job fails. If you want to execute tasks in parallel, you should not select the _wait_ option but instead use the sensor. When configuring the operator, you can can also provide which other Airflow tasks it depends on. If you add a dependency, the task will be executed only after the upstream tasks have been executed successfully. 
+
+- **Sensor**: The sensor can be used to wait for executions to be completed. Similarly to the _wait_ option of the operator, the sensor blocks until the job execution is completed. The sensor can be used to launch several jobs in parallel and wait for their execution to be completed. Please note that the sensor is defined at the job level rather than the execution level. The sensor will wait for the most recent execution to be completed and it will fail the Airflow task if the execution was not successful. 
+
+You can then create the DAG and Hopsworks will generate the Python file. 
+
+#### Write your own DAG 
+
+If you prefer to code the DAGs or you want to edit a DAG built with the builder tool, you can do so. The Airflow DAGs are stored in the _Airflow_ dataset which you can access using the file browser in the project settings.
```

Comment:
You can also edit the DAG is a Jupyter notebook in Hopsworks by opening the python file from the _Airflow_ dataset.